### PR TITLE
Organisation detail page

### PR DESCRIPTION
**Pull request by mirjamverloop**
- Collection detail pages now also show organisation
- Including depiction, description and external link (all optional)

### CODE REVIEW

File: src/store/search.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ export const defaultFacetNames = [
 const filtersFromQf = (qfs) => {
   const filters = {};
 
+  if (!qfs || !qfs[0]) {
```
Comment (rwd) [inline src/store/search.js:25]: Why is this needed?

Comment (mirjamverloop) [reply src/store/search.js:25]: It breaks on organisation pages without a check.
qfs could be null on these pages.

Comment (rwd) [reply src/store/search.js]: I have pushed a change to address (https://github.com/europeana/portal.js/pull/1125/commits/4e640813aa1d1c08c7f775a0cc59fb1504e701bc) this in a different way, returning the fielded filter needed (I think) for the organization URIs.

**Review by lbiedinger — APPROVED** [review]
